### ISSUE TITLE
Holding shift when going to previous item doesn't register for speed

### ISSUE THREAD

**Issue by BryceAG** [phetsims/friction#53]
When you shift tab to one of the books as a previous selection and hold shift, the speed is not affected even though shift should be slowing it down. Releasing shift and repressing it works correctly. In the gif below I show the slow speed, go to the previous book, and then move it again while still pressing shift but it moves faster.

![ezgif com-video-to-gif 2](https://user-images.githubusercontent.com/15327201/33680866-25e78628-da80-11e7-87e3-88be159d8d73.gif)

related to phetsims/QA#75

**Comment by jessegreenberg** [comment]
The keydown event for shift doesn't register when focus moves to  the macro book, so it doesn't 'know' that shift is held down. This is a general problem with keyboard dragging. Can we update keystate in KeyboardDragListener to capture shift on focus event?

**Comment by jessegreenberg** [comment]
For 'shift' key, we can do something like this in the keyup listener:
```js
      // if the shift key is down when we navigate to the object, add it to the keystate because it won't be added until the next keydown event
      if ( event.keyCode === Input.KEY_TAB ) {
        if ( event.shiftKey ) {

          // add 'shift' to the keystate until it is released again
          self.keyState.push( {
            keyDown: true,
            keyCode: Input.KEY_SHIFT,
            timeDown: 0 // in ms
          } );
        }
      }
```

But this only works for shift, and @phet-steele pointed out cases where we want the state of the keyboard to propagate through navigation (probably).

**Comment by jessegreenberg** [comment]
I think for this release we should proceed with the snippet in https://github.com/phetsims/friction/issues/53#issuecomment-350134297, and in releases where we have other hotkeys (like BASE), we should look into a long term fix.

**Comment by jessegreenberg** [comment]
KeyboardDragListener does not exist in master so cherry-pick of https://github.com/phetsims/scenery/commit/736f586a8ab900ee43e94353b1932bfd5d6c949a will not be possible but it should be easy enough to apply change to old KeyboardDragHandler.

**Comment by phet-steele** [comment]
Looks to be working.

**Comment by samreid** [comment]
There are still TODOs marked for this issue, discovered during https://github.com/phetsims/chipper/issues/946

**Comment by jessegreenberg** [comment]
This TODO was promoted a more general issue, I updated the link referenced in the comment, see https://github.com/phetsims/scenery/issues/1054.
